tests/boulder-integration.sh

```diff
@@ -20,6 +20,18 @@ else
   readlink="readlink"
 fi
 
+cleanup_and_exit() {
+    EXIT_STATUS=$?
+    if SERVER_STILL_RUNNING=`ps -q $python_server_pid -o pid=`
```

Comment:
@bmw I have made the modification that you noted. It is on a separate PR (#4075) due to an "indescribably horrible" merge conflict the previous branch was faced with as a result of having previously merged from master